app/assets/v2/js/quadraticlands/mission/diplomacy/room.js

```diff
@@ -0,0 +1,343 @@
+document.addEventListener('DOMContentLoaded', function() {
+
+  console.debug('DIPLOMACY ROOM');
+
+  
+
+  // show arrownav when #trollbox is in visible area
+  // hide arrownav when #trollbox is not in visible area
+  const arrownav = document.getElementById('arrownav');
+  var observer = new IntersectionObserver(function(entries) {
+    if(entries[0].isIntersecting === true)
+      arrownav.classList.remove("hide");
+    else
+      arrownav.classList.add("hide");
+  }, { threshold: [0.5] });
+  observer.observe(document.querySelector("#trollbox"));
+
+
+
+
+  document.refresh_page = function(url){
+    var keys = [/*'.diplomacy-room-members', '.diplomacyvouchebar', */'.entries']
+    $.get(url, function(response){
```

Comment:
@thelostone-mc heres the second place im stuck ; Getting the page to repaint without page load on vouch.

it works for `.entries` but not for the other two commented out items in the array